src/api/app.js

```diff
@@ -47,5 +70,13 @@ export default {
       logos[optionNumbers[2]],
       logos[optionNumbers[3]]
     ]
+  },
+  isValidArray: function (value) {
+    if (!Array.isArray) {
```

Comment:
`isArray` support seems good enough so I don't think it needs to be polyfilled.

Reply:
Okay, I was thinking of browser support but yeah I will change it.